src/use-css.hook.js

```diff
@@ -0,0 +1,48 @@
+import {useEffect, useState} from 'react'
+import {Scoped} from './scoped.component.js'
+import {styleTags, incrementCounter, getCurrentCounter, transformCss} from './style-element-utils.js'
+
+export function useCss(css, overrideNamespace) {
+  const [kremlingId, setKremlingId] = useState(getCurrentCounter())
+  const isPostCss = css && css.id && css.styles
+  const namespace = overrideNamespace || (isPostCss && isPostCss.namespace) || Scoped.defaultNamespace
```

Comment:
if the `css` object has `id` and `styles` props, then `isPostCss` will equal `css.styles` - looks like that might break the next line when we grab teh `isPostCss.namespace`

Reply:
good catch -- updated.